Chap_API_Server.tex

```diff
@@ -2462,7 +2462,7 @@ \subsection{\code{pmix_server_job_control_fn_t}}
                              const pmix_proc_t *requestor,
                              const pmix_proc_t targets[], size_t ntargets,
                              const pmix_info_t directives[], size_t ndirs,
-                             pmix_info_cbfunc_t cbfunc, void *cbdata)
+                             pmix_op_cbfunc_t cbfunc, void *cbdata)
```

Comment:
Is this signature change necessary for Python or was it always supposed to be `pmix_op_cbfunc_t`?

The description in the text says it should be `pmix_op_cbfunc_t`, but the code n PMIx master the signature of these two functoins look like [this](https://github.com/pmix/pmix/blob/master/include/pmix_server.h#L348-L358):
```
typedef pmix_status_t (*pmix_server_job_control_fn_t)(const pmix_proc_t *requestor,
                                                      const pmix_proc_t targets[], size_t ntargets,
                                                      const pmix_info_t directives[], size_t ndirs,
                                                      pmix_info_cbfunc_t cbfunc, void *cbdata);
typedef pmix_status_t (*pmix_server_monitor_fn_t)(const pmix_proc_t *requestor,
                                                  const pmix_info_t *monitor, pmix_status_t error,
                                                  const pmix_info_t directives[], size_t ndirs,
                                                  pmix_info_cbfunc_t cbfunc, void *cbdata);
```
The [RFC 15](https://github.com/pmix/RFCs/blob/master/RFC0015.md) also has it as an `pmix_info_cbfunc_t` which makes me think that the description in the standard is incorrect. But I don't know if there was a deeper reason why this needed to change.

Reply:
This is strange - I initially changed it because I thought the description said `op_cbfunc`. However, I just reset that commit out - and now I can't find any reference to the `op_cbfunc` for those functions any more. I agree that we should leave it as-is, and I guess I was just reading it all wrong. Thanks for pointing it out.